packages/react-server/src/render/render.tsx

```diff
@@ -102,12 +101,14 @@ export function createRender(render: RenderFunction, options: Options = {}) {
       ctx.set(Header.ContentType, 'text/html');
       ctx.body = response;
     } catch (error) {
-      logger.error(error);
+      const errorMessage = `React SSR Error:\n${error.stack || error.message}`;
+
+      logger.log(errorMessage);
+      ctx.status = StatusCode.InternalServerError;
```

Comment:
Koa's being silly. If I don't set this status to 500, then `ctx.throw` actually wouldn't throw a 500, I was getting 404s